sway-lib-core/src/ops.sw

```diff
@@ -1258,3 +1298,19 @@ pub fn ok_str_eq() {
     assert("" != "a");
     assert("a" != "b");
 }
+
+fn flags() -> u64 {
+    asm() {
+        flag
+    }
+}
+
+fn panic_on_overflow_enabled() -> bool {
```

Comment:
This doesn't actuall panic and is just a predicate.

Call it `is_overflow_enabled` instead or include the panic in the procedure.

Reply:
it is not "panic on (overflow enabled)", it is "(panic on overflow) enabled". I suppose it is a tad bit confusing but the flag  behavior is described as "panic on overflow" everywhere else now, and all other functions use it, so i believe this would be the correct name following the naming convention of declarative names ("enabled" over "is_on") that are followed, atleast in sway applications. Is this not the convention for sway?

Reply:
Ive renamed it in this pr, but theres another function already in the std lib now with the old name of panic_on_overflow_enabled, which needs to be changed in another PR now